README.md

```diff
@@ -919,10 +920,40 @@ Also see [push notifications](#push-notifications)
 * [Nikka](https://github.com/JustaLab/Nikka) - A super simple Networking wrapper that supports many JSON libraries, Futures and Rx :large_orange_diamond: ⌚
 
 #### Email
-
```

Comment:
I added a space between the email tag and its items. Sorry for making this a complicated pull request.